modules/jaeger-config-sampling.adoc

```diff
@@ -0,0 +1,72 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-sampling_{context}"]
+= Jaeger sampling configuration options
+
+Note that while all traces are generated, only a few are sampled.  Sampling a trace marks the trace for further processing and storage.
+
+[NOTE]
+====
+This is not relevant if a trace was started by the Istio proxy as the sampling decision is made there. The Jaeger sampling decisions are only relevant when you are using the Jaeger tracer (client).
```

Comment:
The Jaeger sampling decision is only relevant when the trace is **started** by an application using the Jaeger tracer.